zips/draft-coinholder-directed-grants.md

```diff
@@ -0,0 +1,259 @@
+
+#### Collecting Votes
+
+The shielded voting protocol has a registration window corresponding to a defined block range. Eligibility is determined by Orchard notes created within this block range.
+
+The Administrator MUST choose a registration window of at least 10 days.
+
+The Administrator MUST publish voting guidance to coinholders. This guidance SHOULD include advising participants to move funds out of any wallet used for voting before entering a seed phrase into the Coin Voting 2.0 application, in order to reduce risk if the third-party application is compromised.
+
+Votes MUST be submitted to a BFT voting chain operated by at least four independent Voting Authorities. The rationale for this threshold is that if at least two thirds of Voting Authorities are honest, vote suppression or manipulation becomes infeasible.
+
+#### Counting Votes
+
+The Administrator MUST collect and publish finalized tallies.
+
+TBD: specify how shielded voting is counted.
+
+#### Vote Auditing
+
+Independent auditors MAY verify the results using the published election data.
+
+TBD: specify how shielded voting is audited.
+
+### Transparent Voting
+
+#### Collecting Votes
+
+Before accepting votes, the Administrator MUST publish an Orchard-only IVK designated for receiving vote messages and payments. The corresponding FVK MUST NOT be published.
+
+To create a transparent vote, Voters:
+
+- Create a signed message indicating their vote and transparent address using ECDSA over secp256k1 for the signature.
+- Send the signed message along with a payment of at least 1 ZEC to the address associated with the published IVK.
+
+TBD: finish the specification, including but not limited to:
+- The signing protocol (just specifying a signature algorithm is insufficient).
+- The address derivation to be used.
+- The structure of transaction to be created (what kinds of outputs, how many, to what recipients, with what values).
+
+#### Counting Votes
+
+TBD: specify how transparent votes are counted.
+
+The Administrator MUST ignore submissions:
+
+- with payments below 1 ZEC.
+- where the Voter moves the submitted funds before voting concludes, as specified in the Quarterly Timeline below.
+
+TBD: clarify specification of "voting concludes"; if you mean "at the end of the Coinholder Vote on Proposals phase" then say this instead of relying on the indirect definition.
+
+#### Vote Auditing
+
+The Administrator MUST publish IVK-based audit data and all received vote messages in order to allow verification.
+
+TBD: specify how transparent votes are verified.
+
+## Voting Process
+
+The Administrator MUST publish relevant parameters for each voting method for each cycle, and provide sufficient information for independent auditing.
+
+TBD: specify the rest of the overall voting process.
```

Comment:
Fill out this TBD. This should include:
- How to choose the parameters for each voting method 
- How the set of validators whose responses will be considered authoritative for determining the outcome of the vote is chosen
- Some of the timeline should probably be in this section - this should direct the administrator with respect to how to set the registration window, etc.
- Anything else that requires a choice by the administrator resulting from the completed specifications of the individual shielded and transparent voting mechanism sections.
- How the set of grants being voted on is translated into a set of individual elections.